### ISSUE TITLE
You can't copy text on a form

### ISSUE THREAD

**Issue by Jaguaras** [cryptpad/cryptpad#1046]
### Description

I've created a form and I want that users could copy a specific part of a text. But it looks like it is not possible to do so.

### Steps to reproduce

1. Create a simple form, add simple text in the description.
2. Click on preview form and try to select any text on a form.

### Expected behavior

Users should be able to copy any text in a form or at least there should be function to make text copyable.

### What opreating system are you using?

Apple macOS

### What browsers are you seeing the problem on?

Google Chrome

### Extensions

Metamask

### Version

5.3.0

**Comment by yflory** [comment]
Fixed with a1dd33a. It will be included in CryptPad v5.4
